pdata/pprofile/generated_location.go

```diff
@@ -70,9 +70,9 @@ func (ms Location) SetAddress(v uint64) {
 	ms.orig.Address = v
 }
 
-// Line returns the Line associated with this Location.
-func (ms Location) Line() LineSlice {
-	return newLineSlice(&ms.orig.Line, ms.state)
+// Lines returns the Lines associated with this Location.
+func (ms Location) Lines() LineSlice {
```

Comment:
This is a breaking change so the release note should either be breaking or deprecation (if we add deprecation notices). If it is breaking then let's list the breaking changes on the release note

Reply:
The pprofile changelog entry is marked as breaking.
I've added all the changes in the notes.